charts/fluentd-elasticsearch/templates/configmaps.yaml

```diff
@@ -542,7 +542,7 @@ data:
       suppress_type_name "#{ENV['OUTPUT_SUPPRESS_TYPE_NAME']}"
 {{- end }}
 {{- if .Values.elasticsearch.buffer.enabled }}
-      <buffer>
+      <buffer "#{ENV['OUTPUT_BUFFER_CHUNK_KEYS']}">
```

Comment:
not sure if empty quotes are a problem and a `{{ if .Values.elasticsearch.buffer.chunkKeys }}` should be used here?
have you tested it without setting "chunkKeys" value?

Reply:
Yes, I have perfomred test without chunkKeys and everything seems to work.

Reply:
I added condition as you suggested to avoid confusion.